DEVEL.md

```diff
@@ -64,6 +64,10 @@ Optionally, your device plugin may also implement the
 before they are sent to `kubelet`. To see an example, refer to the FPGA
 plugin which implements this interface to annotate its responses.
 
+In case you want to implement the whole allocation functionality in your
+device plugin, you can implement the optional `deviceplugin.Allocator`
+interface.
```

Comment:
fixed

Reply:
Related to the coverage, there is one line which is too difficult for the amount of work required to get covered inside gpu_plugin.go. Other than that and main(), it is 100%. If you want to increase the gpu_plugin.go coverage, add some sort of a test for main() or refactor that so that parts of it can be tested.